spec/spec_params.rb

```diff
@@ -64,10 +64,12 @@ module OptimizelySpec
       'percentageIncluded' => 10_000,
       'variations' => [{
         'key' => 'control',
-        'id' => '111128'
+        'id' => '111128',
+        'featureEnabled' => true
```

Comment:
Please put 'featureEnabled' prop only with variations, that belong to some experiment, that is attached to some feature flag. In other words, the variation should also have variables node

Reply:
Discussed.